Apps/AdminWebClient/src/ClientApp/src/models/patientData.ts

```diff
@@ -1,11 +1,13 @@
 import Address from "@/models/address";
 import { StringISODate } from "@/models/dateWrapper";
+import Name from "@/models/Name";
 
 export default interface PatientData {
     hdid: string;
     personalhealthnumber: string;
-    firstname: string;
```

Comment:
This is AdminWebClient and it calls v2 so I don't think there should be any changes to this code.